app/controllers/shared/api.php

```diff
@@ -238,6 +264,28 @@
     }
 
     $route = $utopia->match($request);
+
+    $useCache = $route->getLabel('cache', false);
+    if ($useCache) {
+            $key = md5($request->getURI() . $request->getServer('query_string'));
+            $cache = new Cache(new Filesystem(APP_STORAGE_CACHE . DIRECTORY_SEPARATOR . 'app-' . $project->getId()));
+            $data = $response->getPayload();
+        if (!empty($data)) {
+            $cacheLog = $dbForProject->getDocument('cache', $key);
```

Comment:
authorization skip might be needed here?

Reply:
fixed